### DISCUSSION TITLE
About the evaluation metrics

### DISCUSSION THREAD

**Discussion by ali-ihsan-gullu** [christianbirchler-org/sdc-testing-competition#16] [Q&A] [answered]
I have a question about the 'Evaluation.' For the competition, the metrics currently provided are 'time budget (1),' 'fault detection (2),' and 'diversity (3).' In the final evaluation among the teams, are these metrics considered with equal weight? For example, my test-selector performs very well on metric (2), but lags behind another team for metric (1). Will there be a weighting between these, or will they be considered separately? Could you please provide a more detailed explanation of these metrics? Thank you very much.

**Comment by spanichella** [comment, marked as answer]
Hello @vatozZ , for the metrics we are using as reference the competition at SBFT 2024 https://sbft24.github.io/tools/.
However, the weighting of the metrics will be disclosed only after submission. What we can say is that the composition of the dataset we will use for the evaluation will influence the weighting.
